### ISSUE TITLE
Problem with loading config file

### ISSUE THREAD

**Issue by MBThaumatec** [Zodiase/meteor-mdl#26]
Hi,
I was trying to change theme without success. I've got json file: `zodiase-mdl.json` with text in it:

```
{
  "theme": {
    "primary": "deep_orange",
    "accent": "amber"
  }
}
```

I was placing this file anywhere in my project with no luck. After any change I was doing `meteor reset`.

Please point me where to put this file if you have such structure:

```
YourMeteorApp
    .meteor
    app
        client
        lib
        private
        public
        server         
```

I see in your code that you are referencing to `process.cwd()` and that could be the problem.

```
▶ meteor run            
[[[[[                                         
~/Gits/somedir/YourMeteorApp
]]]]]

=> Started proxy.                             
=> Started MongoDB.                           
=> Started your app.                          

=> App running at: http://localhost:3000/
```

No console log that settings were found.

**Comment by Zodiase** [comment]
Did you try putting it at `YourMeteorApp/zodiase-mdl.json`?

**Comment by MBThaumatec** [comment]
Yes, of course. That's `process.cwd()` when you run local package. But it seems to be not when running package from atmosphere

Meteor v1.2.1
Node v0.12.4

**Comment by Zodiase** [comment]
Yep now I can see the issue here as well. Only when the package is added from atmosphere... Now I'm stuck at how to do further testing and eventually come up with a fix. Most likely `process.cwd()` is different when added from atmosphere and the current release of the package doesn't print that info out. I don't really want to push another release just to check `process.cwd()` LOL. Let me think of something...

**Comment by Zodiase** [comment]
I think what I have to do is use `Package.registerBuildPlugin([options])` the same way https://github.com/huttonr/bootstrap3/ is doing.

**Comment by Zodiase** [comment]
Should be fixed in the latest release `1.0.6_5`. Let me know if not.

Unfortunately the fix also broke backward compatibility. Please refer to the README file for updates.

**Comment by MBThaumatec** [comment]
This is working perfectly now, thanks.
